api/src/database/subscription.ts

```diff
@@ -13,6 +13,16 @@ export const getSubscriptions = async (channelAddress: string): Promise<Subscrip
 	return MongoDbService.getDocuments<Subscription>(collectionName, query);
 };
 
+export const getSubscriptionsByAuthorization = async(channelAddress: string, isAuthorized: boolean): Promise<Subscription[] | null> => {
```

Comment:
type should `isAuthorized?: boolean `

in addition: have you check if all will be received if no isAuthorized property is set?

Reply:
Ok just checked it... If I don't provide is-authorized query parameter no subscriptions will be returned but all subscriptions shall be returned.